### PR TITLE
feat: EC2 role action

### LINKED ISSUE #82: EC2 role arn action
Create an action that takes as an input the arn of an EC2 instance. Then using the arn, figure out the role arn of the EC2 instance (you should be able to get this information from the [describe instances API](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html)). Once you have the role ARN, please output that.

### PR DESCRIPTION
**Pull request by kloading**
Closes #82

### CODE REVIEW

**Comment by gauravphoenix** [comment]
right now unit test cases are failing as the PR was created from a fork and we use coveralls for unit test coverage. I am going to fix it by removing coveralls running during these unit tests as we already have an independent github action that publishes unit test overage.

**Comment by gauravphoenix** [comment]
When you make  IAM API call to get the policy from the role, the role name is required and not the arn e.g. `aws iam get-role --role-name roleName`

so what we are really interested in is the actual role name under which the ec2 instance is running. Currently, the action is returning the profile arn e.g. `arn:aws:iam::123456789012:role/roleName` but what is must return is `roleName`. 

At the same time, we also need the role ARN so that we can do integration with cloudsplaining. As such, I think the action should return the following output- 

```json
{
"roleName": "foo",
"roleArn": "arn:aws:iam::123456789012:role/foofoo"

}
```

Finally, note in the above output, we are returning the role arn and not the instance profile arn. 
The ec2 describe instance API returns the instance profile arn e.g. `arn:aws:iam::123456789012:instance-profile/foofoo` but we want role arn e.g. `arn:aws:iam::123456789012:role/foofo`

Fortunately, we don't have to make another AWS IAM API call to get the IAM role, we can just replace `instance-profile` with `role` and it should be just fine. 

In summary, following changes are required- 

1. Return both role name and role arn
2. replace `instance-profile` with `role` to get the role arn.

**Comment by kloading** [comment]
@gauravphoenix Addressed the required changes with latest commit

**Review by gauravphoenix — APPROVED** [review]
